Sources/Commands/PackageCommands/EditCommands.swift

```diff
@@ -32,16 +32,16 @@ extension SwiftPackageCommand {
         @Option(help: "Create or use the checkout at this path")
         var path: AbsolutePath?
 
-        @Argument(help: "The name of the package to edit")
-        var packageName: String
+        @Argument(help: "The identity of the package to edit")
+        var packageIdentity: String
```

Comment:
This ends up being a breaking change. We probably want to add a deprecated and/or hidden `packageName` flag that simply sets the `packageIdentity` so callers don't break when they upgrade.

Reply:
Ah, good catch! Thank you!